### ISSUE TITLE
Bug: bash_completion.install attempting to write to `/etc/bash_completion.d`

### ISSUE THREAD

**Issue by jasonkarns** [Homebrew/homebrew-core#5206]
```sh
$ cd `brew --repo` && git show HEAD
e5274c0
$ brew doctor 
Your system is ready to brew.
```

I'm attempting to update the git-imerge formula. A version bump to 1.0 was already begun: #4986 but was closed prematurely.

I am attempting to remove the conditional around the bash_completion install, since bash completion is now included in git-imerge releases. (The conditional was only installing bash_completion for --head installs, because for a time, there was not a git-imerge release that included completion.)

However, when the conditional is removed and the completion install is attempted, I receive an error:

```sh
install git-imerge /usr/local/Cellar/git-imerge/1.0.0/bin
mkdir: /etc/bash_completion.d: Operation not permitted
make: *** [install] Error 1
```

The formula as it appears when manifesting the error: https://gist.github.com/anonymous/86d4ab735dffae590e6b20da0a00906e

I've also pushed my work to my own fork: https://github.com/jasonkarns/homebrew-core/tree/git-imerge-1.0

(The only change was to remove the conditional, such that the install block is:

```ruby
def install
  bin.mkpath
  system "make", "install", "PREFIX=#{prefix}"
  bash_completion.install "git-imerge.bashcomplete"
end
```

I don't believe this is an issue with the formula, because the bash_completion line looks like virtually every other bash_completion.install command in other core formula.

https://gist.github.com/0b7bbfeb9c05ea33e8e8d8cb0a01a185

Looking for some guidance on how to debug this or if it's an issue with homebrew's bash_completion.install itself. (Should I open an issue on homebrew/brew?)

**Comment by zmwangx** [comment]
I could be wrong, but last time I looked at this it has nothing to do with `bash_completion.install` at all (try removing that line, I bet you'll get the same problem). It's the Makefile writing to `/etc/bash_completion.d`.

**Comment by zmwangx** [comment]
Fixed for you in #5211.

**Comment by zmwangx** [comment]
And closing as not our bug.
